packages/blunderbuss/__snapshots__/blunderbuss.js

```diff
@@ -1,59 +1,65 @@
 exports['Blunderbuss issue tests assigns opened issues with no assignees 1'] = {
-  'assignees': [
-    'issues1'
+  "assignees": [
```

Comment:
Please add `__snapshots__` to `.eslintignore` and `.prettierignore` to avoid these :)